src/stores/Object.js

```diff
@@ -221,6 +221,29 @@ class ObjectStore {
     return this.writeTokens[objectId];
   });
 
+  @action.bound
+  CopyObjectPermission = flow(function * ({libraryId, objectId}) {
+    const meta = yield Fabric.GetContentObjectMetadata({
```

Comment:
_do not_ ever pull all metadata on arbitrary objects

use `select` and `remove` to specify exactly what you want returned

Metadata can be huge, and this is pulling meta for every object in the list.

Reply:
btw, is any of this metadata or owner information available in the info already pulled for the list?

Reply:
No owner information available in the ContentBrowser. When it loads the objects, it pulls the name, title, display_title, and sources

Reply:
Ok, you should also check owner first, and if that is true you can return immediately. You should also wrap the metadata call in try/catch because the requesting user may not have access to the private meta of the object.